collect/collect.go

```diff
@@ -108,6 +117,13 @@ func (i *InMemCollector) Start() error {
 	i.Metrics.Register("trace_send_dropped", "counter")
 	i.Metrics.Register("trace_send_has_root", "counter")
 	i.Metrics.Register("trace_send_no_root", "counter")
+
+	i.Metrics.Register("trace_send_shutdown_total", "counter")
```

Comment:
Since these only occur on Shutdown, maybe we don't want them as metrics at all? We could attach this data to traces and avoid creating a bunch of metrics that are usually zero.